src/folding/hypernova/nimfs.rs

```diff
@@ -304,13 +315,9 @@ impl<C: CurveGroup> NIMFS<C> {
         }
 
         // Verify the interactive part of the sumcheck
-        let sumcheck_subclaim = <PolyIOP<C::ScalarField> as SumCheck<C::ScalarField>>::verify(
-            sum_v_j_gamma,
-            &proof.sc_proof,
-            &vp_aux_info,
-            transcript,
-        )
-        .unwrap();
+        let sumcheck_subclaim =
+            IOPSumCheck::<C, T>::verify(sum_v_j_gamma, &proof.sc_proof, &vp_aux_info, transcript)
+                .unwrap();
```

Comment:
Error handling missing here too?? 
If this is instead an invariant check (which I don't think it is) you should use `expect` instead.

Reply:
ditto as above.

Reply:
[Fix](https://github.com/privacy-scaling-explorations/folding-schemes/pull/40/commits/59431ecd6e88b724fea6c4cdd52e46959eb4e1a0)